src/main/resources/db/changelog/changes/08-insert-users-roles-table.yaml

```
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users-roles-table
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet